install_scripts/ruby.sh

```diff
@@ -6,11 +6,7 @@ echo "Setting up ruby environment"
 PACKAGES="imagemagick libreadline-dev libyaml-dev libsqlite3-dev libqtwebkit-dev nodejs zlib1g-dev libsqlite3-dev nodejs redis-server"
 sudo apt-get -y install $PACKAGES
```

Comment:
these guys seem a little hidden in the "ruby" scriptie -- maybe put in geo_concerns?